R/bootstrap.R

```diff
@@ -827,3 +887,14 @@ renv_bootstrap_user_dir_impl <- function() {
 
 }
 
+renv_bootstrap_version_is_dev <- function(version) {
```

Comment:
It still needs slightly different code because it's slightly different test to figure out if we're using a sha or a version number. So IMO, it's not worth it.